make/migrations/postgresql/0010_1.9.0_schema.up.sql

```diff
@@ -0,0 +1,10 @@
+/* add table for CVE whitelist */
+CREATE TABLE cve_whitelist (
+    id SERIAL PRIMARY KEY NOT NULL,
+    project_id int,
```

Comment:
why not share the 'scope' that label is using? 

If we define 0(project_id) is the system level, should we propagate it?

Reply:
There's no need to apply the "scope" in label we are using, because there's only one whitelist.